Core/src/Propagator/SympyStepper.cpp

```diff
@@ -120,8 +126,16 @@ void SympyStepper::transportCovarianceToBound(
       freeToBoundCorrection);
 }
 
+Result<double> SympyStepper::step(State& state, Direction propDir,
+                                  const IVolumeMaterial* material) const {
+  (void)material;
```

Comment:
Nitpick:

```suggestion
  static_cast<void>(material);
```

for new code, ideally.